tests/mmptest/src/ExtensionTests.cs

```diff
@@ -20,8 +20,10 @@ public void TodayExtension_SmokeTest ()
 
 			MMPTests.RunMMPTest (tmpDir =>
 			{
-				string testPath = Path.Combine (TI.FindSourceDirectory (), @"Today/TodayExtensionTest.csproj");
-				TI.BuildProject (testPath, isUnified: true);
+				TI.CopyDirectory (Path.Combine (TI.FindSourceDirectory (), @"Today"), tmpDir);
```

Comment:
The extension tests were building in place, which was lazy and wrong of me. Let's copy them to temp.

We also need this since we added a variable to substitute in today.